avocado/utils/linux_modules.py

```diff
@@ -208,34 +208,107 @@ def get_loaded_modules():
         return [astring.to_text(_.split(b' ', 1)[0]) for _ in proc_modules]
 
 
-def check_kernel_config(config_name):
+def check_kernel_config(config_name, session=None):
     """
     Reports the configuration of $config_name of the current kernel
 
     :param config_name: Name of kernel config to search
+    :param session: guest session, command is run on host if None
     :type config_name: str
     :return: Config status in running kernel (NOT_SET, BUILTIN, MODULE)
     :rtype: :class:`ModuleConfig`
     """
 
-    kernel_version = platform.uname()[2]
+    def check_local_kernel_config(config_name):
+        """
+        Reports the configuration of $config_name of the current kernel
+
+        :param config_name: Name of kernel config to search
+        :return: Config status in running kernel (NOT_SET, BUILTIN, MODULE)
+        :rtype: :class:`ModuleConfig`
+        """
+        kernel_version = platform.uname()[2]
+
+        config_file = '/boot/config-' + kernel_version
```

Comment:
This can be  `config_file = '/boot/config-' + platform.uname()[2]`
As not using the kernel_version any more in this function. 
So avoid extra variable initialization.